docs/apps.rst

```diff
@@ -51,6 +51,8 @@ Applications
 
 .. automodule:: apps.demo
 
+.. automodule:: draw
```

Comment:
`Draw` with a capital D. rename `apps/draw.py` to `apps/Draw.py` to match

Reply:
ok